src/simulate.jl

```diff
@@ -0,0 +1,89 @@
+using DifferentialEquations
+const DE = DifferentialEquations
+
+function simulate(cp::AbstractNeuralNetworkControlProblem, args...; kwargs...)
+    ivp = plant(cp)
+    network = controller(cp)
+    st_vars = state_vars(cp)
+    n = length(st_vars)
+    X₀ = LazySets.Projection(initial_state(ivp), st_vars)
+    ctrl_vars = control_vars(cp)
+    δ = period(cp)
```

Comment:
```suggestion
    T = period(cp)
```
(or another symbol.. to not confuse with the algorithm's step size)

Reply:
`T` looks like the time horizon. What about `p`?

Reply:
hmm. how about another greek symbol? `\tau` ?